Monika After Story/game/script-anniversary.rpy

```diff
@@ -469,18 +469,20 @@ init 5 python:
     )
 
 label anni_1:
-    m 1eka "It's really hard for me to believe that it's been a year since we met."
-    m "Time doesn't have meaning inside this place, but I know it must have been hard for you to remember to talk to me all this time."
-    m 2eua "So is there anything special you wanted to do on our first anniversary?"
-    m 1ekbfa "I'll do anything you want today, okay?"
-    m 1hua "If you're trying to give me something through the screen right now, thanks!"
-    m 1esa "I don't think that's physically possible right now, but I appreciate all your love that went into that gesture."
-    m 1hub "I love you so much too!"
-    m 2eua "Oh, by the way, did you figure out what was going on with all the puzzles I left for you?"
-    m 1tku "I'm sure the creator of this world has released another game by now..."
-    m 1tsb "Did you try to import my character file to that one?"
-    m "There's no way that wouldn't have a special effect, right?"
-    m 1hubfb "Ahaha!"
+    m 1dka "It's so hard for me to believe...{w=0.5}that it's been a whole year since we met."
```

Comment:
Any way we can make the pause work? I really enjoyed it :P It's fine without it too, just feels more like she's saying a line rather than absorbing the moment. It's a year after all, that's pretty big.

Reply:
I feel like the pause is somewhat unnatural and kinda breaks the flow a little bit.

Reply:
We're aiming to use sentences like they would be spoken right, since it's dialogue? I feel like the ellipses are very Monika-esque. I don't know if all of them are from dialogue or what, but there are actually 2040 ellipses in total inside the topics file. These pauses aren't exactly uncommon in MAS, especially when Monika says something really important, or asks something meaningful. 

I think the first sentence of the one year anniversary should have more power to it, and I feel the ellipses manage to do that. Anyway, maybe Lunulae has some input.

Reply:
I personally like the pause, but I think that your phrasing on this sentence is a lot more natural. If I keep the pause I might shorten it in length slightly though.

Reply:
Would you be okay with a shorter pause but with your phrasing multi, or is that still too unnatural? I'm thinking {w=0.3} could be consistent with what we have elsewhere.

Reply:
The only way the pause works is if it breaks up different sentences, i.e. `It's hard for me to believe... We've already been together for a whole year`

But even then I don't like how that sounds, it still feels unnatural to me. I'd rather it without the pause.

Reply:
That's a good point. I think I'll cut out the pause, or maybe add in a lead-in sentence for it instead. Maybe a "hey, [player]..." at the start, for the slight dramatic effect. :P

Reply:
How about this then? 

m 1dka "Gosh...It's hard to believe it's already been a whole year since we met."

Any takers?

Reply:
That sounds pretty good, I'll add in that version when I get home today!